### PR TITLE
Fix check table from multiple database connection

### PR DESCRIPTION
**Pull request by sefirosweb**
Hello I fixed the check of table with multiple database connection

Related with the topic:
https://github.com/scrumble-nl/laravel-model-ts-type/issues/27

Best!

### CODE REVIEW

File: src/Support/Generators/DatabasePropertyGenerator.php [hunk 1]
```diff
@@ -20,12 +20,13 @@ public function getPropertyDefinition(Model $model): array
         $propertyDefinition = [];
         $table = $model->getTable();
         $connection = DB::connection($model->getConnection()->getName());
+        $conectionName = $model->getConnection()->getName();
```
Comment (RicoClark) [inline src/Support/Generators/DatabasePropertyGenerator.php]: Please type `connection` with 2 n's. 
Also, please initialize the variable above the $connection initialization, and re-use the variable

**Comment by RicoClark** [comment]
Thank you for your pull request, please check my comment

**Comment by sefirosweb** [comment]
Hi @RicoClark i don't see your comments,

I need to do something?

Best,

**Comment by RicoClark** [comment]
Excuse me @sefirosweb, now it should be visible

**Comment by sefirosweb** [comment]
Done!

**Review by Luukdewaaier — APPROVED** [review]

**Pull request merged**